it/controllers/HelloWorldControllerISpec.scala

```diff
@@ -0,0 +1,35 @@
+
+package controllers
+
+import helpers.BaseIntegrationSpec
+import play.api.http.Status._
+import play.api.test.FakeRequest
+import uk.gov.hmrc.play.test.UnitSpec
+
+class HelloWorldControllerISpec extends UnitSpec with BaseIntegrationSpec {
+
+  "Calling the helloWorld action" when {
+
+    "authenticated" should {
```

Comment:
Can you call this something slightly more informative please? Something like `"User is authenticated"` would be better, same for the unit test for HelloWorldController